network.go

```diff
@@ -82,10 +82,16 @@ type DNSInfo struct {
 	Options  []string
```

Comment:
This is regarding your commit message. Physical interfaces were being skipped in the second scan

Reply:
Yes but this means we could not retrieve the Link type in that case (if needed obviously).